modules/web-console/e2e/testcafe/mocks/agentTasks.js

```diff
@@ -14,13 +14,38 @@
  * limitations under the License.
  */
 
+import _ from 'lodash';
+const base64js = require('base64-js');
```

Comment:
@Klaster1 Could you confirm that `require` should be used here? I expected to see `import`.

Reply:
Nodejs [has native base64 support](https://stackoverflow.com/a/6182519/333777).

Reply:
Changed to import.

Reply:
Reworked to Node.js Buffer.